tests/Integration/PrestaShopBundle/Test/WebTestCase.php

```diff
@@ -74,18 +75,23 @@ public function setUp()
         $employeeMock->id_profile = 1;
 
         $contextMock = $this->getMockBuilder(Context::class)
-            ->setMethods(array('getTranslator', 'getBaseURL'))
+            ->setMethods(array('getTranslator', 'getContext'))
             ->disableOriginalConstructor()
             ->getMock();
 
         $contextMock->method('getTranslator')
-            ->will($this->returnValue($this->translator));
+            ->will(self::returnValue($this->translator));
+
+        $contextMock->method('getContext')
```

Comment:
I try to cover the case where Context call itself: so wrong situation but very popular in Legacy code sadly :(